app/javascript/application.js

```diff
@@ -1,3 +1,8 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
+import '@hotwired/turbo-rails'
 import 'datatables'
 import 'stupidtable'
+
+// Forms by default will not be submitted with turbo unless the
+// data-turbo="true" attribute is present. See: https://github.com/hotwired/turbo/pull/419
+Turbo.setFormMode("optin")
```

Comment:
This sounds like something that's not true in the latest Turbo, probably have to either revisit or add a comment to delete this eventually